WordPress/gutenberg
# Blocks: Split frontend block styles into `block.scss` files

This PR builds on top of #1418 and splits all the blocks' styles into two files 

 - `block.scss` for styles shared with the frontend
 - `style.scss` for styles only loaded in the editor.

This should allow themes to support the gutenberg blocks without any change. The styles can be overriden by themes.

**Testing instructions**

Try the following blocks and ensure they're showing up properly on the frontend and the backend:

 - button
 - cover image
 - pullquote
 - quote
 - separator
